charter/index.md

```diff
@@ -0,0 +1,228 @@
+---
+title: Font and Text Community Group Charter
+---
+This charter:  {{ page.url | remove: "index.html" | prepend: site.url }}
+
+Feedback about this document:
+    https://github.com/font-text/font-text.github.io/issues
```

Comment:
After we migrate to the [new repo](https://github.com/w3c/font-text-cg), this may need to be updated.

Reply:
Done.